lib/pengine/unpack.c

```diff
@@ -1060,7 +1075,8 @@ unpack_handle_remote_attrs(node_t *this_node, xmlNode *state, pe_working_set_t *
     }
 
     if (crm_is_true(g_hash_table_lookup(this_node->details->attrs, "maintenance")) ||
-        (rsc && !is_set(rsc->flags, pe_rsc_managed))) {
+        (rsc && !is_set(rsc->flags, pe_rsc_managed)) ||
+        (rsc && rsc->container && !is_set(rsc->container->flags, pe_rsc_managed))) {
```

Comment:
yes, a leftover - but probably better to invest in an additional test now instead of thinking why it is there in a year or so ...